ffcx/__init__.py

```diff
@@ -13,24 +13,11 @@
 import logging
 import pkg_resources
 
-from FIAT import supported_elements
-
 __version__ = pkg_resources.get_distribution("fenics-ffcx").version
 
-
 logging.basicConfig()
 logger = logging.getLogger("ffcx")
 logging.captureWarnings(capture=True)
 
-# Import main function, entry point to script
-from ffcx.main import main  # noqa: F401
-
 # Import default parameters
 from ffcx.parameters import default_parameters  # noqa: F401
-
-# Duplicate list of supported elements from FIAT and emove elements from
-# list that we don't support or don't trust
-supported_elements = sorted(supported_elements.keys())
```

Comment:
Should this be handed elsewhere?

Reply:
It is handled. It was done twice in a weird way. FIAT has supported elements which clearly says which elements it supports, we should not monkey patch it.

Supported elements by FFCx are stated in `fiatinterface.py` as `supported_families`.